subcommands/targets/offline-update.go

```diff
@@ -84,6 +87,15 @@ func doOfflineUpdate(cmd *cobra.Command, args []string) {
 	fmt.Println("Successfully refreshed and downloaded TUF metadata")
 
 	if !ouTufOnly {
+		if !isDstDirClean(dstDir) {
+			fmt.Println("Destination directory already has update data")
+			if !ouAllowMultipleTargets {
+				subcommands.DieNotNil(errors.New(
+					"Please provide a clean destination directory or re-run with --allow-multiple-targets to add a new target to a directory which already has update data.\n" +
+						"Notice that multiple targets in the same directory is only supported in LmP >= v92."))
```

Comment:
We usually use multi-line messages for this:
```suggestion
				subcommands.DieNotNil(errors.New(`
Provide a clean destination directory, or re-run with --allow-multiple-targets to add a new target to a directory which already has update data.
Notice that multiple targets in the same directory is only supported in LmP >= v92.`))
```

If you don't want an extra newline (usually it is OK) - move the opening backtick above to the line start.

Also, it is kind of strange to see a message which starts with `ERROR: Please`: it is not clear what was an error.
IIUC @kprosise would recommend to have no `Please` above, so I also included that suggestion.
Still, some error needs to be shown _after_ `ERROR:` and _before_ giving an instruction how to tackle it.